### PR TITLE
backport: trivial 2024 02 28

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
## What was done?
Trivial Backports

## How Has This Been Tested?
Built locally; did not run tests or review

## Checklist:
  _Go over all the following points, and put an `x` in all the boxes that apply._
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation
- [ ] I have assigned this pull request to a milestone _(for repository code-owners and collaborators only)_

### CODE REVIEW

File: doc/descriptors.md [hunk 1]
```diff
@@ -11,13 +11,16 @@ Supporting RPCs are:
...
-- `importmulti` takes as input descriptors to import into the wallet
+- `importmulti` takes as input descriptors to import into a legacy wallet
   (since v0.18).
 - `generatetodescriptor` takes as input a descriptor and generates coins to it
   (`regtest` only, since v0.19).
 - `utxoupdatepsbt` takes as input descriptors to add information to the psbt
   (since v0.19).
-- `createmultisig` and `addmultisigaddress` return descriptors as well (since v0.20)
+- `createmultisig` and `addmultisigaddress` return descriptors as well (since v0.20).
+- `importdescriptors` takes as input descriptors to import into a descriptor wallet
```
Comment (UdjinM6) [inline doc/descriptors.md:21]: 25925: this rpc was introduced in 16528 which will be backported in not-yet-merged #5579

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
mostly ok but imo 25925 is blocked by #5579

**Comment by PastaPastaPasta** [comment]
25925 dropped for now; watching https://gitlab.com/dashpay/dash/-/pipelines/1194837720 for CI.

**Comment by DashCoreAutoGuix** [comment]
Guix Automation has began to build this PR tagged as v20.1.0-devpr5909.e3e4a955. A new comment will be made when the image is pushed.

**Comment by UdjinM6** [comment]
Triggered GH Guix build because of 25788, just in case

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**